core/addenda/propertychain/TemporalRelationChain.rdf

```diff
@@ -0,0 +1,200 @@
+	<!ENTITY owl "http://www.w3.org/2002/07/owl#">
+	<!ENTITY rdf "http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+	<!ENTITY rdfs "http://www.w3.org/2000/01/rdf-schema#">
+	<!ENTITY skos "http://www.w3.org/2004/02/skos/core#">
+	<!ENTITY xsd "http://www.w3.org/2001/XMLSchema#">
+]>
+<rdf:RDF xml:base="https://spec.industrialontologies.org/core/propertychain/TemporalRelationChain/"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+     xmlns:terms="http://purl.org/dc/terms/"
+     xmlns:AnnotationVocabulary="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/">
+    <owl:Ontology rdf:about="https://spec.industrialontologies.org/core/propertychain/TemporalRelationChain/">
+        <owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/202301/core/meta/AnnotationVocabulary/"/>
+        <terms:abstract xml:lang="en">The IOF Core Ontology contains a set of object properties that hold either between two temporal interval or two process instances, following the temporal relations proposed by Allen. This non-normative supplemental file contains a set of property chains which infer new temporal relation between two instances of temporal interval or process based on their temporal relation to a third instance of temporal interval or process. For example, if i is related to i&apos; by r and i&apos; is related to i&apos;&apos; by r&apos; where i, i&apos;, i&apos;&apos; are all instances of either temporal interval or process  and r, r&apos; are some temporal relations then i is related to i&apos;&apos; by (r o r&apos;), where (r o r&apos;) is the composition of r and r&apos;. In this file, only those composition of temporal relations are captured which yield another basic temporal relation and not a disjunction of basic temporal relations.</terms:abstract>
+        <terms:contributor>Jim Logan, No Magic, Inc.</terms:contributor>
+        <terms:contributor>Milos Drobnjakovic, National Institute of Standards and Technology (NIST)</terms:contributor>
+        <terms:creator>Arkopaul Sarkar, École Nationale d’Ingénieurs de Tarbes (ENIT)</terms:creator>
+        <terms:license rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">http://opensource.org/licenses/MIT</terms:license>
+        <terms:publisher xml:lang="en">Industrial Ontology Foundry</terms:publisher>
+        <terms:title>Property chains for inferring compositions of Allen&apos;s temporal relations</terms:title>
+        <rdfs:label xml:lang="en">Property chains for inferring compositions of Allen&apos;s temporal relations</rdfs:label>
+        <AnnotationVocabulary:copyright>Copyright (c) 2022, 2023 Open Applications Group</AnnotationVocabulary:copyright>
+    </owl:Ontology>
```

Comment:
It would be good to add a change note to say what changed in this version.

Reply:
For the initial version, I guess we can skip. Just realized this was first release of this addenda.